test/jest/domain.test.js

```diff
@@ -0,0 +1,128 @@
+/* @flow */
+import {
+    getPayPalDomain,
+    getPayPalAPIDomain,
+    insertMockSDKScript
+} from '../../src';
+
+
+beforeEach(() => {
+    __ENV__ = 'test';
+});
+
+describe(`getPayPalDomain function test cases`, () => {
+    it('should successfully get the production paypal domain', () => {
+        __ENV__ = 'production';
+        const expectedPayPalDomain = 'https://www.paypal.com';
+
+        if (getPayPalDomain() !== expectedPayPalDomain) {
+            throw new Error(`Expected paypal domain to be ${ expectedPayPalDomain }, got ${ getPayPalDomain() }`);
```

Comment:
`expect` should be fine to use for Jest assertions. I think we just need to tell eslint about the Jest globals.

Reply:
Great job on these tests @gabrielo91! 💯

Reply:
Yes, I can avoid eslint throw errors, however, the build still fails because of @flow  (I guess, I can check that if we definitely want to use `expect`

![image](https://user-images.githubusercontent.com/17114924/114111831-92cc8b80-98a0-11eb-8a50-4b250dcb7363.png)

Reply:
I found a way, but I don't know how bad is it, since is not clear to me what `flow`  does exactly... Said that... We  can add 
`.*\.test\.js` on .flowconfig and the build works... however, and again.. not sure if is the right way... another one according to what I read, is to add an extra plugin for babel, but such option seems less attractive

Reply:
I have another commit with that approach (jest-expect) and the build working, so we can switch between them and chose the right one ... regarding the coverage, I think that more files are had in the account now, so we need to adjust that (It's not because test coverage was reduced)

Reply:
I wonder if we can use this babel plugin to get jest working with flow: 
- https://babeljs.io/docs/en/babel-plugin-transform-flow-strip-types.html. 
- https://www.grzegorowski.com/jest-tests-flow-type